### PR TITLE
Create test_as_marcao.mmd

### PR DESCRIPTION
**Pull request by regineheberlein**
Document the testing and updating cycles during as_marcao testing

### CODE REVIEW

File: tools/as_marcao/test_as_marcao.mmd [hunk 1]
```diff
@@ -0,0 +1,47 @@
+sequenceDiagram;
```
Comment (sandbergja) [inline tools/as_marcao/test_as_marcao.mmd:1]: Could we add ```mermaid to get the diagram to display in-line?  Also, could we add a title for accessibility?

````suggestion
```mermaid
---
title: Testing process for as_marcao plugin
---
sequenceDiagram;
````

File: tools/as_marcao/test_as_marcao.mmd [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+sftp-staging1->>sftp-staging1: SUCCEEDS
+Princeton-->>sftp_prod1: migrates server
+Lyrasis-->>aspace-staging: removes as_marcao v.0.5
+Lyrasis-->>aspace-prod: installs as_marcao v.0.5
+Princeton-->>sftp-staging1: updates firewall rules for aspace-prod
+aspace-prod-->>sftp-staging1:runs as_marcao v0.5
+sftp-staging1->>sftp-staging1: fails
+Lyrasis-->>aspace-prod: adds config
+aspace-prod-->>sftp-staging1:runs as_marcao v0.5
+sftp-staging1->>sftp-staging1: SUCCEEDS
```
Comment (sandbergja) [inline tools/as_marcao/test_as_marcao.mmd:47]:
````suggestion
sftp-staging1->>sftp-staging1: SUCCEEDS
```
````

**Review by sandbergja — CHANGES_REQUESTED** [review]
Thanks for documenting this process, @regineheberlein !  Just a few small questions that should help it display better in Github.

File: tools/as_marcao/test_as_marcao.mmd (hunk 1 above)
[the remark this answers is not in the document]
Comment (regineheberlein) [reply tools/as_marcao/test_as_marcao.mmd:5]: Should it? I'm using the VS tomoyukim Mermaid Editor extension, and it's not recognizing .`md` as a mermaid file. But maybe it needs to be that for github?

[the remark this answers is not in the document]
Comment (regineheberlein) [reply tools/as_marcao/test_as_marcao.mmd:5]: (Also, thank you! :-) )

[the remark this answers is not in the document]
Comment (sandbergja) [reply tools/as_marcao/test_as_marcao.mmd:5]: Oh, if that messes it up for you in vs code, best to keep it as is.

**Review by sandbergja — APPROVED** [review]
Thanks, @regineheberlein !

**Pull request merged**